src/tngcli/cli.py

```diff
@@ -188,7 +188,7 @@ def dispatch(args):
     
     #monitor subcommand
     elif args.subparser_name == 'monitor':
-        sel_args = [args.target_list, args.service_list, args.metric_list, args.vnf_uuid, args.vdu_uuid]
+        sel_args = [args.target_list, args.service_list, args.metric_list, args.vnf_uuid, args.vdu_uuid, args.metric_name]
```

Comment:
Hi Panos. We are trying to be conform pep8 python standard, so line lengths can't exceed 80 chars. Can you split this one up please?

Reply:
Hi Thomas, I think that its ok now